SeedVaultSimulator/src/main/AndroidManifest.xml

```diff
@@ -64,11 +64,11 @@
             android:exported="true"
             android:permission="com.solanamobile.seedvault.ACCESS_SEED_VAULT"
             android:targetActivity=".ui.seeddetail.SeedDetailActivity">
-            <intent-filter android:priority="100" android:order="-100">
```

Comment:
It turns out that negative `order` values are not honored. It's not critical, since no disambiguation should be required, but this helps when:
- an app (incorrectly) holds both the `ACCESS_SEED_VAULT` and `ACCESS_SEED_VAULT_PRIVILEGED` permissions
- the app does not use the new `SeedVault.resolveComponentForIntent` helper, which always prefers the privileged `<intent-filter>` Activity